### DISCUSSION TITLE
How to implement auto reveal on buffer change?

### DISCUSSION THREAD

**Discussion by k0va1** [nvim-tree/nvim-tree.lua#1816] [Q&A] [answered]
I'm using telescope and I want to auto reveal file(highlight current) when I selected one. Now I'm doing it manually calling `:NvimTreeFindFile`. Is it possible to do it automatically?

**Comment by alex-courtis** [comment, marked as answer]
See `:help nvim-tree.update_focused_file`

Please note there is currently a bug with this behaviour with telescope: #1815

**Reply by k0va1** [reply]
thank u!

**Reply by unphased** [reply]
Suggest chaging default from false to true because this is the kind of feature that will turn a lot of users away by not having. 

But I also get that some minority of users will be extremely annoyed by automated behavior if it is undesired.
